Library/Formula/bazel.rb

```diff
@@ -21,6 +22,7 @@ def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
 
     system "./compile.sh"
+    system "./output/bazel build scripts:bash_completion"
```

Comment:
Please separate the arguments: `system "./output/bazel", "build", "scripts:bash_completion"`.